admin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bens Admin Page</title>
+    <link rel='stylesheet' type='text/css' href='css/normalise.css'>
+    <link rel='stylesheet' type='text/css' href='css/admin.css'>
+</head>
+
+<body>
+    <header>
+        <h1>Admin Page</h1>
+        <h2>Welcome Ben</h2>
+        <h3>About Me</h3>
```

Comment:
If there is a chance you'll have different subsections of this page would you not want to break out the 'About me' header to be in it's own section, rather than in the overall header? For example for a portfolio or contacts section, would you add a new 'header' area, or just put it at the top of it's section?